### PR TITLE
Add short PipeWire section to Linux installation instructions

### LINKED ISSUE #287: [Feature] Flatpak installation guide
**Is your feature request related to a problem? Please describe.**
Installation on Linux involves/involved compilation which might not be easy. Now there are multiple ways to install Jamulus on Linux, one if via Flathub: https://github.com/jamulussoftware/jamulus/issues/803 
**Describe the solution you'd like**

Maybe: an installation guide for Jamulus via Flatpak from Flathub?

### PR DESCRIPTION
**Pull request by paulmenzel**

### CODE REVIEW

**Comment by paulmenzel** [comment]
Sorry, no idea, why the remark-lint test fails. It shows issues with all files, and the summary(?) – maybe of new warnings/errors – refers to line 30 and 32, which I do not touch.

File: wiki/en/en-Installation-for-Linux.md [hunk 1]
```diff
@@ -15,25 +15,35 @@ Make sure you read the [Getting Started](Getting-Started) page.
...
 1. Install the package: `sudo apt install /path/to/{{ site.download_file_names.deb-gui }}`.
-1. Since Jamulus needs the JACK server, you have to install it too. We recommend to use `QjackCtl` to configure JACK. You can install it via `sudo apt-get install qjackctl`
+1. Since Jamulus needs the JACK server, you have to install it too. We recommend to use `QjackCtl` to configure JACK. You can install it via `sudo apt install qjackctl`
 
 Afterwards you should look at the "[Set up your sound card](#set-up-your-sound-card)" step on this page.
 
-**Note:** Jamulus (with GUI) is also [included in Debian Bullseye (testing)](https://packages.debian.org/bullseye/jamulus) thanks to [mirabilos](https://github.com/mirabilos). If you already use Bullseye, just install Jamulus via `sudo apt-get install jamulus`.
+**Note:** Jamulus (with GUI) is also [included in Debian Bullseye (testing)](https://packages.debian.org/bullseye/jamulus) thanks to [mirabilos](https://github.com/mirabilos). If you already use Bullseye, just install Jamulus via `sudo apt install jamulus`.
+
+### PipeWire
```
Comment (ann0see) [inline wiki/en/en-Installation-for-Linux.md:38]: I think the section of setting up pipewire should go into the community knowledge base (see the _posts dir).

**Review by ann0see — CHANGES_REQUESTED** [review]

**Comment by ann0see** [comment]
I think we can remove the remark lint since it just fails and doesn't give too much benefit.

File: wiki/en/en-Installation-for-Linux.md (hunk 1 above)
Comment (paulmenzel) [reply wiki/en/en-Installation-for-Linux.md:38]: Thank you for your comment. I intentionally left out how to set it up. Do you mean my comment about how PipeWire worked better on my system than the jackd?

Comment (ann0see) [reply wiki/en/en-Installation-for-Linux.md:38]: I'd totally remove the pipewire reference here and give a link to another page with more explanation. Keep this even shorter (or remove it completely and give a link to the Knowledge Base).

File: wiki/en/en-Installation-for-Linux.md [hunk 2]
```diff
@@ -15,25 +15,35 @@ Make sure you read the [Getting Started](Getting-Started) page.
 Although we don't provide official packages for every Linux distribution, you might find Jamulus in the package manager of your distro. Have a look at this table:
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/jamulus.svg)](https://repology.org/project/jamulus/versions)
-
+P
```
Comment (ann0see) [inline wiki/en/en-Installation-for-Linux.md]: Could you please remove this?

**Review by ann0see — CHANGES_REQUESTED** [review]

File: wiki/en/en-Installation-for-Linux.md (hunk 2 above)
Comment (paulmenzel) [reply wiki/en/en-Installation-for-Linux.md]: Sorry about that, and thank you for spotting this.

**Comment by gilgongo** [comment]
Agree this is one for the [Knowledge Base](https://jamulus.io/kb/index.html). Feel free to create a page on that for this.

**Pull request closed without merging**